server/athenian/api/internal/miners/github/deployment.py

```diff
@@ -2022,3 +2024,132 @@ async def load_jira_issues_for_deployments(deployments: pd.DataFrame,
                 prs[pri] = np.sort(np.array(release_keys, dtype="S"))
 
     return issues
+
+
+async def hide_anomal_first_deployments(
+    account: int,
+    meta_ids: Sequence[int],
+    deployment_facts: pd.DataFrame,
+    pdb: Database,
+    mdb: Database,
+    threshold: float = 100.0,
+) -> None:
+    """Hide the anomal first deployments by deleting their facts from pdb."""
+    log = logging.getLogger(f"{__name__}.hide_first_deployments")
```

Comment:
The log name doesn't agree with the function name

Reply:
changed to `log = logging.getLogger(f"{__name__}.hide_outlier_first_deployments")`
(also for the private function, I think it's more useful to have the name of the higher level function for the logger there)